modules/camera/camera_picamera2.py

```diff
@@ -24,23 +25,28 @@ class CameraPiCamera2(base_camera.BaseCameraDevice):
     __create_key = object()
 
     @classmethod
-    def create(cls, width: int, height: int) -> "tuple[True, CameraPiCamera2] | tuple[False, None]":
+    def create(
+        cls, width: int, height: int, config: PiCameraConfig = None
+    ) -> "tuple[True, CameraPiCamera2] | tuple[False, None]":
         """
         Picamera2 Camera.
 
         width: Width of the camera.
         height: Height of the camera.
-
+        config (PiCameraConfig): Configuration object
         Return: Success, camera object.
         """
         try:
             camera = picamera2.Picamera2()
 
-            config = camera.create_still_configuration(
+            camera_config = camera.create_still_configuration(
                 {"size": (width, height), "format": "RGB888"}
             )
-            camera.configure(config)
+            camera.configure(camera_config)
```

Comment:
Can you change this portion to create_preview_configuration(). If you don't want to make a new name, you can also combine the whole thing together into a line